integrations/statuspage/.port/resources/port-app-config.yaml

```diff
@@ -0,0 +1,124 @@
+            url: .url
+            allow_page_subscribers: .allow_page_subscribers
+            allow_incident_subscribers: .allow_incident_subscribers
+            allow_email_subscribers: .allow_email_subscribers
+            allow_sms_subscribers: .allow_sms_subscribers
+            allow_rss_atom_feeds: .allow_rss_atom_feeds
+            allow_webhook_subscribers: .allow_webhook_subscribers
+            time_zone: .time_zone
+            createdAt: .created_at
+            updatedAt: .updated_at
+  - kind: component_group
+    selector:
+      query: "true"
+    port:
+      entity:
+        mappings:
+          identifier: .id
+          title: .name
+          blueprint: '"statuspageComponentGroup"'
+          properties:
+            description: .description
+            position: .position
+            createdAt: .created_at
+            updatedAt: .updated_at
+          relations:
+            statuspage: .page_id
+  - kind: component
+    selector:
+      query: ".group == false"
+    port:
+      entity:
+        mappings:
+          identifier: .id
+          title: .name
+          blueprint: '"statuspageComponent"'
+          properties:
+            description: .description
+            position: .position
+            status: .status
+            showcase: .showcase
+            only_show_if_degraded: .only_show_if_degraded
+            startDate: '.start_date | if . == null then null else (strptime("%Y-%m-%d") | todateiso8601) end'
+            createdAt: .created_at
+            updatedAt: .updated_at
+          relations:
+            componentGroup: .group_id
+            statuspage: .page_id
+  - kind: incident
+    selector:
+      query: "true"
+    port:
+      entity:
+        mappings:
+          identifier: .id
+          title: .name
+          blueprint: '"statuspageIncident"'
+          properties:
+            status: .status
+            impact: .impact
+            createdAt: .created_at
+            updatedAt: .updated_at
+            startedAt: .started_at
+            resolvedAt: .resolved_at
+            shortlink: .shortlink
+            scheduled_for: .scheduled_for
+            scheduled_until: .scheduled_until
+            scheduled_remind_prior: .scheduled_remind_prior
+            scheduled_reminded_at: .scheduled_reminded_at
+            impact_override: .impact_override
+            scheduled_auto_in_progress: .scheduled_auto_in_progress
+            scheduled_auto_completed: .scheduled_auto_completed
+            metadata: .metadata
+            reminder_intervals: .reminder_intervals
+            postmortem_body: .postmortem_body
+            postmortem_body_last_updated_at: .postmortem_body_last_updated_at
+            postmortem_ignored: .postmortem_ignored
+            postmortem_published_at: .postmortem_published_at
+            postmortem_notified_subscribers: .postmortem_notified_subscribers
+            postmortem_notified_twitter: .postmortem_notified_twitter
+            pageId: .page_id
```

Comment:
This is unnecessary since we already have the relation, please also remove it from the blueprint

Reply:
I have removed the property now